client/next.config.js

```diff
@@ -1,31 +1,54 @@
 /** @type {import('next').NextConfig} */
 
-const gatewayUrl = process.env.NEXT_PUBLIC_GATEWAY_URI || "http://localhost:8030"
-const faucetUrl = process.env.NEXT_PUBLIC_FAUCET_URI || "http://localhost:3001"
+const gatewayUrl = process.env.NEXT_PUBLIC_GATEWAY_URI || 'http://localhost:8030'
+const faucetUrl = process.env.NEXT_PUBLIC_FAUCET_URI || 'http://localhost:3001'
 
 module.exports = {
   reactStrictMode: true,
   webpack: (config, { isServer, webpack }) => {
     // Fixes npm packages that depend on `fs` module
     if (!isServer) {
-      config.resolve.fallback.fs = false;
+      config.resolve.fallback.fs = false
     }
     config.plugins.push(
-			new webpack.IgnorePlugin({
-				resourceRegExp: /^electron$/,
-			})
-		);
+      new webpack.IgnorePlugin({
+        resourceRegExp: /^electron$/
+      })
+    )
+    config.module.rules.push({
+      test: /\.svg$/,
+      issuer: /\.tsx?$/,
+      use: [
+        {
+          loader: '@svgr/webpack',
```

Comment:
This [loader](https://www.npmjs.com/package/@svgr/webpack) allows us to render SVGs directly in the html and we can change their [styling](https://github.com/nevermined-io/defi-marketplace/pull/385/files#diff-f5b16ae0eba6d343931cb21d7e827c5805bec43202eb527760597f8be9debebbR232-R235)